lib/routes/routePUT.js

```diff
@@ -143,10 +143,6 @@ export default function routePUT(request, response, log, utapi) {
                 return routesUtils.responseNoBody(errors.MissingContentLength,
                     null, response, 411, log);
             }
-            if (Number.isNaN(request.parsedContentLength)) {
-                return routesUtils.responseNoBody(errors.InvalidArgument,
```

Comment:
Looking at the error message, I'd understand that the first `if`was actually the erroneous one, and should check for the presence of the ContentLength header. Do we really want to remove this difference between invalid Argument and Missing Content length ?